backends-clickhouse/src/main/scala/io/glutenproject/utils/CHExpressionUtil.scala

```diff
@@ -16,8 +16,54 @@
  */
 package io.glutenproject.utils
 
-import io.glutenproject.expression.ExpressionMappings._
+import io.glutenproject.expression.ExpressionMappings.{CRC32, SHA2, _}
+import io.glutenproject.utils.CHExpressionUtil.STRING_TYPE
 
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.catalyst.expressions.{Expression, GetJsonObject, Literal}
+
+trait FunctionValidator extends Logging {
+  final val DATE_TYPE = "date"
```

Comment:
move to `UnixTimeStampValidator` ?

Reply:
Not sure whether it will be used in other validators later, put it here may be OK

Reply:
It's better indicate in the special XXXValidator but not in the base trait FunctionValidator.